anchore_engine/db/entities/policy_engine.py

```diff
@@ -943,6 +954,19 @@ def __repr__(self):
             self.__class__, self.name, self.created_at
         )
 
+    @property
+    def normalized_id(self):
+        """normalized_id will inspect the in coming external
+        references and return a cve id in the case of a single
+        match against vulndb information.
+        """
+        res = _search_dict("source", "CVE ID", self.references)
+        if len(res) == 2:
+            name = "CVE-" + res["url"].split("name=")[1]
+        else:
+            name = self.name
+        return name
```

Comment:
This just returns the first instance of `CVE ID` object in references. Additional logic is required to compute that there is only one CVE reference before returning it as the normalized ID. Change this to something like 

```suggestion
        res = _get_one_or_none("source", "CVE ID", self.references)
        if res and res.get("url"):
            # regex match might be better, going with split for now. also making assumptions about link structure, need to verify
            parts = res.get("url").split("name=", 1)
            if parts and len(parts) > 1:
                return "CVE-" + parts[1]

        return self.name
```